### ISSUE TITLE
TestComposeProcessor

### ISSUE THREAD

**Issue by bossenti** [apache/streampipes#2771]

**Comment by pambrocio** [comment]
please assign this to me

**Comment by pambrocio** [comment]
build successful
```
[INFO] 
[INFO] Apache StreamPipes ................................. SUCCESS [  1.898 s]
[INFO] streampipes-archetype-extensions-jvm ............... SUCCESS [  0.835 s]
[INFO] streampipes-archetype-pe-sinks-flink ............... SUCCESS [  0.071 s]
[INFO] streampipes-archetype-pe-processors-flink .......... SUCCESS [  0.074 s]
[INFO] StreamPipes Commons ................................ SUCCESS [  2.820 s]
[INFO] streampipes-model-shared ........................... SUCCESS [  1.000 s]
[INFO] streampipes-vocabulary ............................. SUCCESS [  1.611 s]
[INFO] streampipes-model .................................. SUCCESS [  4.570 s]
[INFO] streampipes-dataformat ............................. SUCCESS [  0.911 s]
[INFO] streampipes-messaging .............................. SUCCESS [  0.886 s]
[INFO] streampipes-client-api ............................. SUCCESS [  1.096 s]
[INFO] streampipes-dataformat-json ........................ SUCCESS [  0.870 s]
[INFO] streampipes-dataformat-cbor ........................ SUCCESS [  0.877 s]
[INFO] streampipes-dataformat-fst ......................... SUCCESS [  0.868 s]
[INFO] streampipes-security-jwt ........................... SUCCESS [  0.884 s]
[INFO] streampipes-model-client ........................... SUCCESS [  0.649 s]
[INFO] streampipes-extensions-api ......................... SUCCESS [  1.530 s]
[INFO] streampipes-test-utils ............................. SUCCESS [  1.116 s]
[INFO] streampipes-serializers-json ....................... SUCCESS [  1.268 s]
[INFO] streampipes-client ................................. SUCCESS [  1.105 s]
[INFO] streampipes-measurement-units ...................... SUCCESS [  2.632 s]
[INFO] streampipes-sdk .................................... SUCCESS [  4.174 s]
[INFO] streampipes-connect-shared ......................... SUCCESS [  2.184 s]
[INFO] streampipes-service-discovery-api .................. SUCCESS [  0.925 s]
[INFO] streampipes-dataformat-smile ....................... SUCCESS [  0.858 s]
[INFO] streampipes-storage-api ............................ SUCCESS [  1.161 s]
[INFO] streampipes-storage-couchdb ........................ SUCCESS [  2.899 s]
[INFO] streampipes-storage-management ..................... SUCCESS [  1.045 s]
[INFO] streampipes-user-management ........................ SUCCESS [  1.636 s]
[INFO] streampipes-mail ................................... SUCCESS [  1.309 s]
[INFO] streampipes-resource-management .................... SUCCESS [  2.337 s]
[INFO] streampipes-service-discovery ...................... SUCCESS [  0.902 s]
[INFO] streampipes-pipeline-management .................... SUCCESS [  3.345 s]
[INFO] streampipes-connect-management ..................... SUCCESS [  2.633 s]
[INFO] streampipes-extensions-management .................. SUCCESS [  2.684 s]
[INFO] streampipes-data-explorer-commons .................. SUCCESS [  2.539 s]
[INFO] streampipes-data-explorer .......................... SUCCESS [  2.001 s]
[INFO] streampipes-data-export ............................ SUCCESS [  1.460 s]
[INFO] streampipes-service-base ........................... SUCCESS [  1.042 s]
[INFO] streampipes-rest-shared ............................ SUCCESS [  1.039 s]
[INFO] streampipes-rest-extensions ........................ SUCCESS [  2.021 s]
[INFO] streampipes-service-extensions ..................... SUCCESS [  1.405 s]
[INFO] Apache StreamPipes Maven Plugin .................... SUCCESS [  4.040 s]
[INFO] streampipes-extensions ............................. SUCCESS [  0.009 s]
[INFO] streampipes-connectors-mqtt ........................ SUCCESS [  1.076 s]
[INFO] streampipes-pipeline-elements-shared ............... SUCCESS [  0.723 s]
[INFO] streampipes-connect-adapters ....................... SUCCESS [  0.797 s]
[INFO] streampipes-connect-adapters-iiot .................. SUCCESS [  2.159 s]
[INFO] streampipes-wrapper ................................ SUCCESS [  1.284 s]
[INFO] StreamPipes Wrapper for Standalone Pipeline Element Implementations SUCCESS [  1.376 s]
[INFO] streampipes-connectors-influx ...................... SUCCESS [  1.244 s]
[INFO] streampipes-messaging-kafka ........................ SUCCESS [  0.988 s]
[INFO] streampipes-connectors-kafka ....................... SUCCESS [  1.200 s]
[INFO] streampipes-messaging-nats ......................... SUCCESS [  0.846 s]
[INFO] streampipes-connectors-nats ........................ SUCCESS [  1.061 s]
[INFO] streampipes-connectors-opcua ....................... SUCCESS [  2.826 s]
[INFO] streampipes-connectors-pulsar ...................... SUCCESS [  2.592 s]
[INFO] streampipes-connectors-rocketmq .................... SUCCESS [  2.295 s]
[INFO] streampipes-connectors-tubemq ...................... SUCCESS [  1.238 s]
[INFO] streampipes-processors-filters-jvm ................. SUCCESS [  4.251 s]
[INFO] streampipes-connectors-plc ......................... SUCCESS [  1.987 s]
[INFO] streampipes-messaging-mqtt ......................... SUCCESS [  0.965 s]
[INFO] streampipes-messaging-pulsar ....................... SUCCESS [  1.092 s]
[INFO] streampipes-wrapper-siddhi ......................... SUCCESS [  2.883 s]
[INFO] streampipes-processors-filters-siddhi .............. SUCCESS [  1.966 s]
[INFO] streampipes-processors-transformation-jvm .......... SUCCESS [  4.291 s]
[INFO] streampipes-messaging-jms .......................... SUCCESS [  0.883 s]
[INFO] streampipes-sinks-brokers-jvm ...................... SUCCESS [  1.251 s]
[INFO] streampipes-sinks-databases-jvm .................... SUCCESS [  1.623 s]
[INFO] streampipes-sinks-internal-jvm ..................... SUCCESS [  2.109 s]
[INFO] streampipes-sinks-notifications-jvm ................ SUCCESS [  2.559 s]
[INFO] streampipes-processors-enricher-jvm ................ SUCCESS [  1.767 s]
[INFO] streampipes-processors-change-detection-jvm ........ SUCCESS [  1.106 s]
[INFO] streampipes-extensions-all-iiot .................... SUCCESS [  2.016 s]
[INFO] streampipes-processors-image-processing-jvm ........ SUCCESS [  1.353 s]
[INFO] streampipes-processors-text-mining-jvm ............. SUCCESS [  1.224 s]
[INFO] streampipes-processors-geo-jvm ..................... SUCCESS [  1.820 s]
[INFO] streampipes-extensions-all-jvm ..................... SUCCESS [  2.941 s]
[INFO] streampipes-extensions-iiot-minimal ................ SUCCESS [  1.634 s]
[INFO] StreamPipes Integration Tests ...................... SUCCESS [ 27.424 s]
[INFO] streampipes-rest-core-base ......................... SUCCESS [  1.097 s]
[INFO] StreamPipes REST API ............................... SUCCESS [  2.395 s]
[INFO] streampipes-platform-services ...................... SUCCESS [  1.409 s]
[INFO] streampipes-sdk-bundle ............................. SUCCESS [  0.210 s]
[INFO] StreamPipes Core ................................... SUCCESS [  2.744 s]
[INFO] streampipes-service-core-minimal ................... SUCCESS [  1.139 s]
[INFO] streampipes-wrapper-distributed .................... SUCCESS [  1.103 s]
[INFO] StreamPipes Wrapper for Apache Flink ............... SUCCESS [  1.762 s]
[INFO] streampipes-wrapper-kafka-streams .................. SUCCESS [  1.246 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:53 min
[INFO] Finished at: 2024-04-25T09:11:58+08:00
[INFO] ------------------------------------------------------------------------
```

**Comment by pambrocio** [comment]
PR https://github.com/apache/streampipes/pull/2782

**Comment by tenthe** [comment]
Fixed in PR #2375.
